indico/util/user.py

```diff
@@ -153,3 +154,36 @@ def wrapper(*args, **kwargs):
         return fn(*args, **kwargs)
 
     return wrapper
+
+
+def _unify_pending_user_data(decorated_func):
+    @wraps(decorated_func)
+    def _wrapper(user_data):
+        user_data['first_name'] = user_data.pop('firstName')
+        user_data['family_name'] = user_data.pop('familyName')
+        return decorated_func(user_data)
+
+    return _wrapper
+
+
+@_unify_pending_user_data
+def create_pending_user(user_data):
```

Comment:
Can't we use that [here](https://github.com/indico/indico/blob/master/indico/modules/users/util.py#L198)?